offline-election/src/subcommands/staking.rs

```diff
@@ -164,7 +173,9 @@ pub async fn run(client: &Client, opt: Opt, conf: StakingConfig) {
 	}
 
 	// stash key of all wannabe candidates.
-	let mut candidates = get_candidates(client, at).await;
+	let mut candidates = get_candidates_paged(client, at).await;
```

Comment:
is this really enough? `get_voters` is still using `enumerate_map` and this means that we cannot still use safe rpc nodes.